features/page_objects/header_page.py

```diff
@@ -0,0 +1,5 @@
+# BUTTONS
+ADDTOOCART1_BTN = '#tbodyid > div.row > div > a'
```

Comment:
is this button part of the header?
or is part of the product page?

I think you are missing that one. a page where you put all the elements related to the products.

Reply:
and why did you add a number here?